packages/cli/src/platform/auth/show.ts

```diff
@@ -41,9 +45,36 @@ export class Show implements Command {
         `,
       },
     })
+
+    let sensitiveData: { token: string | null } = {
+      token: null,
+    }
```

Comment:
That sounds good! Will do, thanks @jasonkuhrt

Reply:
@jasonkuhrt I actually just realised now that we already get the token earlier on in this function, just a few lines up. Sorry for missing that 🤦:
https://github.com/prisma/prisma/blob/247b4b9d24fc900e503af34cbc9f76b061d216f7/packages/cli/src/platform/auth/show.ts#L19

 So in the end I didn't end up adding `loadOrThrow()`, since I didn't need it anymore. The changes are just as simple though.